content/documentation/pages/4-batch-developer-guides/2-batch/8-data-flow-simple-task-cloudfoundry.md

```diff
@@ -0,0 +1,264 @@
+
+- Using local tools (via SSH or external database provider)
+- Using a Database GUI deployed to Cloud Foundry
+
+### Using local tools (MySQLWorkbench)
+
+```bash
+cf create-service-key task-example-mysql EXTERNAL-ACCESS-KEY
+cf service-key task-example-mysql EXTERNAL-ACCESS-KEY
+```
+
+This should give you back the credentials neccessary to access the database, e.g.:
+
+```
+Getting key EXTERNAL-ACCESS-KEY for service instance task-example-mysql as ghillert@gopivotal.com...
+
+{
+ "hostname": "...",
+ "jdbcUrl": "jdbc:mysql://...",
+ "name": "...",
+ "password": "...",
+ "port": "3306",
+ "uri": "mysql://...",
+ "username": "..."
+}
+```
+
+This should result in a response detailing the access information for the respective database. The result will be different depending on whether the used database service is running internally or whether the service is provided by a third-party. In case of PWS, using ClearDB, we can directly connect to the database as it is a third-party provider.
+
+If you are dealing with an internal service, you may have to create an SSH tunnel via the `cf ssh` command, e.g.:
+
+```bash
+$ cf ssh -L 3306:<host_name>:<port> task-example-mysql
+```
+
+Using the free [MySQLWorkbench](https://www.mysql.com/products/workbench/) you should see the following populated data:
+
+![billrun database results](images/CF-task-standalone-task2-database-result.png)
+
+### Using a Database GUI deployed to Cloud Foundry
+
+Another interesting option to keep an eye on your MySQL instance is to use [PivotalMySQLWeb](https://github.com/pivotal-cf/PivotalMySQLWeb). In a nutshell, you can push PivotalMySQLWeb to your Cloud Foundry space and bind it to your MySql instance allowing you to introspect your MySQL service without having to use local tooling.
+
+Check out the project:
+
+```bash
+$ git clone https://github.com/pivotal-cf/PivotalMySQLWeb.git
+$ cd PivotalMySQLWeb
+```
+
+**IMPORTANT**: Please update the credentials first in `src/main/resources/application-cloud.yml`
+
+Then build the project:
+
+```bash
+./mvnw -DskipTests=true package
+```
+
+Next, update the `manifest.yml` file:
+
+```yml
+applications:
+  - name: pivotal-mysqlweb
+    memory: 1024M
+    instances: 1
+    random-route: true
+    path: ./target/PivotalMySQLWeb-1.0.0-SNAPSHOT.jar
+    services:
+      - task-example-mysql
+    env:
+      JAVA_OPTS: -Djava.security.egd=file:///dev/urandom
+```
+
+**IMPORTANT** specify your MySQL service `task-example-mysql`
+
+In this instance we set the property `random-route` to `true` in order to generate a random URL for the application. Watch the console for a printout of the URL. Push the app to Cloud Foundry:
+
+```bash
+$ cf push
+```
+
+Now you can login into the application an take a look at the table populated by the `billrun` task application.
```

Comment:
We need to tell them the default user name and password.   i.e. Default username = admin Default password = cfmysqlweb